JapaneseFilter/sections/adservers.txt

```diff
@@ -55,6 +55,7 @@
 !
 ! Fc2.com
 ||analyzer*.fc2.com^
+||media5.fc2.com^
```

Comment:
better with `$third-party`

Reply:
If that is better in terms of performance. I don't believe this domain will be accessed as 1st party. I redirected to `https://fc2.com/` when tried.

Reply:
Got it.

Reply:
Could you please provide your email or telegram account? I want to send something)

Reply:
I don't have telegram account. Is tmail00 still alive, or I don't remember but you once suggested an alternative email? I'll send mine, just not want to expose to public eyes.

Reply:
yes, tmail00@yandex.ru is working

Reply:
Sent.

Reply:
Hi, just sayin' thank you!